code/game/objects/items/storage/cases.dm

```diff
@@ -13,6 +13,7 @@
 	throw_speed = 3
 	throw_range = 7
 	var/empty = FALSE
+	var/max_items = 10
```

Comment:
did you mean to use this somewhere?

Reply:
tl;dr it has no use right now
However, I may make it into something for the future but right now it's just a generic small-item storage which is bulky. 
In the future I could make it printable in the lathe for storing small things like circuitboards? Hell, I could add that onto this PR if you like since it'd mesh well with missions to gather small items like puce and chilly peppers
(though I would go and make sure it can ONLY store small items)

Reply:
OH I meant that var in particular, the max_items thing

Reply:
it has no use rn. I just set it to ten in a vain attempt to try and get it's child type (the surgery case) to hold enough items then didn't set it back.
We can blow up this line if you'd like

Reply:
yee blow it up and I'll merge

Reply:
all done